src/main/java/org/terasology/tutorialWorldGenerationBiomes/BiomeRasterizer.java

```diff
@@ -39,13 +39,12 @@ public void initialize() {
     public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         ElevationFacet elevationFacet = chunkRegion.getFacet(ElevationFacet.class);
         SeaLevelFacet seaLevelFacet = chunkRegion.getFacet(SeaLevelFacet.class);
-        for (Vector3i position : chunkRegion.getRegion()) {
-            if (position.y > Math.max(seaLevelFacet.getSeaLevel(), elevationFacet.getWorld(position.x, position.z)) + 10) {
+        for (Vector3ic position : chunkRegion.getRegion()) {
+            if (position.y() > Math.max(seaLevelFacet.getSeaLevel(), elevationFacet.getWorld(position.x(), position.z())) + 10) {
                 biomeRegistry.setBiome(TutorialBiome.SKY, position);
-            } else if (elevationFacet.getWorld(position.x, position.z) + 1 > seaLevelFacet.getSeaLevel()) {
+            } else if (elevationFacet.getWorld(position.x(), position.z()) + 1 > seaLevelFacet.getSeaLevel()) {
                 biomeRegistry.setBiome(TutorialBiome.LAND, position);
-            }
-            else {
+            } else {
```

Comment:
Nice catch!
FYI: IntelliJ has a nice feature to "Reformat/Rearrange Code" that apply code style rules to either a code fragment or complete files (https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html)